.github/workflows/sync-github-issues.yaml

```diff
@@ -0,0 +1,24 @@
+name: Sync Github issues based on scorecards
+
+
+on:
+  workflow_dispatch:
+
+jobs:
+    sync-github-issues:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Sync Github Issues
+              uses: port-labs/port-sender@v0.2.4
+              with:
+                operation_kind: issue_handler
+                port_client_id: ${{ secrets.PORT_CLIENT_ID }}
+                port_client_secret: ${{ secrets.PORT_CLIENT_SECRET }}
+                blueprint: app
+                scorecard: productionReadiness
+                github_api_url: https://api.github.com
+                filter_rule: '{"property": "$team","operator": "containsAny","value": ["AAA"]}'
+                github_token: ${{ secrets.GITHUB_TOKEN }}
+                github_owner: example-organization
+                github_repo: example-repo
```

Comment:
I would add example on how this can be resolved from the environments, instead of manually specifying. I would like to reduce as much friction from the user